package.json

```diff
@@ -3,15 +3,15 @@
   "version": "0.1.112",
   "description": "The Cisco Spark for React library allows developers to easily incorporate Spark functionality into an application.",
   "scripts": {
-    "build": "npm run build:packages && npm run build:bundle",
-    "build:packages": "cross-env NODE_ENV=${NODE_ENV:-production} babel-node --presets=stage-1,es2015 scripts/build-packages.js",
-    "build:package": "cross-env NODE_ENV=${NODE_ENV:-production} babel-node --presets=stage-1,es2015 scripts/build-package.js",
-    "build:bundle": "cross-env NODE_ENV=${NODE_ENV:-production} babel-node --presets=stage-1,es2015 scripts/build-bundle.js",
-    "build:packagejson": "cross-env NODE_ENV=${NODE_ENV:-production} babel-node --presets=stage-1,es2015 scripts/update-package-json.js",
-    "build:widgets": "cross-env NODE_ENV=${NODE_ENV:-production} babel-node --presets=stage-1,es2015 scripts/build-widgets.js",
-    "clean": "npm run clean:snapshots && npm run clean:dist",
+    "build": "cross-env NODE_ENV=${NODE_ENV:-production} ./scripts/build/index.js",
+    "build:all": "npm run build all",
+    "build:package": "npm run build dist",
+    "build:packagejson": "npm run build package.json",
+    "build:widgets": "npm run build widgets",
+    "clean": "npm run clean:snapshots && npm run clean:dist && npm run clean:transpile",
     "clean:snapshots": "rimraf \"packages/**/__snapshots__/\"",
```

Comment:
This is a piece of legacy script when we were still tweaking how we were building snapshots and the jest script couldn't support `-u` for snapshot updates.